R/get_attribution.R

```diff
@@ -130,8 +152,16 @@ get_attribution <- function(uuid = NULL, img = NULL, text = FALSE) {
                   "and were added using the rphylopic R package ver. ",
                   packageVersion("rphylopic"), " (Gearty & Jones, 2023). ",
                   att)
+    # Add permalink?
+    if (permalink) {
+      att <- paste0(att, " Permalink: ", perm, ".")
+    }
     return(message(att))
   }
   # Return data ----------------------------------------------------------
+  # Add permalink?
+  if (permalink) {
+    att$permalink <- perm
```

Comment:
I like the use of a list here, but should the permalink should be at the same level as the individual images? Maybe the images should be a list item (`att$images`) so that they are collectively at the same level, but individually one level down? Therefore, looping through the list won't cause problems for users (because they would instead always loop through `att$images`, which will always be the same length as `uuids`).

Reply:
If you go with my suggested return behavior change, the text paragraph could also be returned as an item at the top level.